tztrout/data.py

```diff
@@ -243,27 +266,22 @@ def _get_latest_tz_names(self, tz):
 
     def _get_tz_identifiers_for_us_zipcode(self, zipcode):
         """Get all the possible identifiers for a given US zip code."""
-        if not isinstance(zipcode, ZipCode):
-            zipcode = ZipCodeDatabase().get(zipcode)
-            if zipcode:
-                zipcode = zipcode[0]
-            else:
-                return
+        if not isinstance(zipcode, Zipcode):
```

Comment:
Do you expect anything other than a Zipcode? If not, let's remove.

Reply:
Done in 79b5bd4